dependencies.R

```diff
@@ -7,5 +7,5 @@ install.packages(
     "gt",
     "tibble"
   )
+  , repos = "http://cran.us.r-project.org"

```

Comment:
Although `http://cran.us.r-project.org` works, we should probably use `https://cloud.r-project.org/` instead, which automatically chooses/redirect to the most appropriate CRAN server, considering your current location.
```suggestion
  , repos = "https://cloud.r-project.org"

```